### PR TITLE
Fetch data from api

### PR DESCRIPTION
**Pull request by piesieomane**
In this Pull Request the following list below was implemented;

- use redux-thunk in the redux store and createAsyncThunk to make calls from the API
- fetch data from the API using Axios

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,23 +1,57 @@
...
+      ],
+    ],
+  };
+});
+export const removeBook = createAsyncThunk(REMOVE_BOOK, async (id) => {
+  await axios.delete(
+    `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/PX6aInUB6II3Dr3TyuaU/books/${id}`,
+  );
+  return { id };
 });
```
Comment (julie-ify) [inline src/redux/books/books.js:57]: - [ ] **[OPTIONAL]** While the books can be deleted from the API, they also need to be removed from the state simultaneously. Right now after deleting a book, it does not disappear from the page, I need to manually refresh the page before the state is updated. Kindly refactor your deletion to update the state immediately.

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @piesieomane,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (piesieomane) [reply src/redux/books/books.js:57]: @julie-ify Hello, thank you for the review, I have done the necessary changes but on my dashboard it says `submit`

Comment (julie-ify) [reply src/redux/books/books.js:57]: Alright @piesieomane it's been approved

Comment (piesieomane) [reply src/redux/books/books.js:57]: @julie-ify ,Thank you ✔

**Pull request merged**